pkgs/applications/video/handbrake/default.nix

```diff
@@ -25,7 +25,7 @@
 , numactl
 , writeText
   # Processing, video codecs, containers
-, ffmpeg-full
+, ffmpeg_4
```

Comment:
Why don't you use ffmpeg5 instead, handbrake [supports it]("https://github.com/HandBrake/HandBrake-contribs/releases).

Reply:
I just went towards updating handbrake alltogeher.

Reply:
If v1.6.1 by default uses ffmpeg5, so should we.